glitch/repr/inter.py

```diff
@@ -90,16 +99,25 @@ def __init__(self, name: str, value: str, has_variable: bool) -> None:
         self.name: str = name
         self.value: str = value
         self.has_variable: bool = has_variable
+        self.attributes: list = []
 
     def __repr__(self) -> str:
         value = repr(self.value).split('\n')[0]
-        name = self.name.split('.')[-1]
-        return f"{name}:{value}"
+        name = self.name
+        if value == "None":
+            attributes = self.attributes
```

Comment:
I would use `self.attributes` in the return instead of defining the local variable.